charts/kafka-operator/templates/operator-deployment-with-webhook.yaml

```diff
@@ -223,14 +238,17 @@ spec:
             - containerPort: {{ .Values.alertManager.port }}
               name: alerts
               protocol: TCP
+            - containerPort: {{ .Values.livenessReadinessProbe.port | default 8081 }}
```

Comment:
The `| default 8081` in this line is duplicating with https://github.com/banzaicloud/koperator/blob/7083c023df51951829184be5cf7691ad741c0859/charts/kafka-operator/values.yaml#L77 

IMO we should just use either one instead of both